src/create-plugin.js

```diff
@@ -2,10 +2,17 @@
 
 import type {FusionPlugin} from './types.js';
 
-export function createPlugin<Deps, Service>(
-  opts: $Exact<FusionPlugin<Deps, Service>>
-): FusionPlugin<Deps, Service> {
+// eslint-disable-next-line
```

Comment:
Nit, could we add the specific rule we're disabling after `eslint-disable-next-line`?